FWCore/ParameterSet/interface/ParameterSetDescriptionFillerBase.h

```diff
@@ -106,7 +105,6 @@ namespace edm {
 
     static const std::string& extendedBaseType(EDAnalyzer const*) { return kExtendedBaseForEDAnalyzer; }
     static const std::string& extendedBaseType(EDProducer const*) { return kExtendedBaseForEDProducer; }
```

Comment:
I think this PR should have removed the `EDProducer` overload as well. On the other hand, I suspect #40466 could already be in (mild, touching adjacent lines) conflict with this PR, so maybe it is easier to leave the cleanup to #40466 or later PR.